app/javascript/mastodon/actions/tags_typed.ts

```diff
@@ -1,12 +1,39 @@
+import { createAction } from '@reduxjs/toolkit';
+
 import {
   apiGetTag,
   apiFollowTag,
   apiUnfollowTag,
   apiFeatureTag,
   apiUnfeatureTag,
+  apiGetFollowedTags,
 } from 'mastodon/api/tags';
 import { createDataLoadingThunk } from 'mastodon/store/typed_functions';
 
+export const fetchFollowedHashtags = createDataLoadingThunk(
+  'tags/fetch-followed',
+  async ({
+    context,
+    limit,
+    next,
+  }: {
+    context: 'full' | 'sidebar';
```

Comment:
I think it might be simpler to fetch a full page of tags from the API and limit to displaying the first 4 in the sidebar, if it means avoiding keeping track of "where will this be displayed" in actions/reducers. That's what we do with lists.